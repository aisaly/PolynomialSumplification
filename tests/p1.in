3x*5+4
