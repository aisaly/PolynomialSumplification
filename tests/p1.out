(1+2)
Not implemented
Not implemented
